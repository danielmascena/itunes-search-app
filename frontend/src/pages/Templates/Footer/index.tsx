import * as React from "react";

const Footer = () => (
    <footer>
        <p>Copyright</p>
    </footer>
);

export default Footer;