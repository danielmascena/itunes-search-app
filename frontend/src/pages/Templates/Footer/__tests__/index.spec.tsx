import React from "react";
import renderer from 'react-test-renderer';

import Footer from "../index";

test("should renders the Footer component", () => {
    expect(renderer.create(<Footer />).toJSON()).toMatchSnapshot();
});