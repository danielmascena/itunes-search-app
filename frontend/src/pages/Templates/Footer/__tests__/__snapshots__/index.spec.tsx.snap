// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders the Footer component 1`] = `
<footer>
  <p>
    Copyright Â© 
    <a
      href="https://github.com/danielmascena"
      title="author"
    >
      Daniel Mascena
    </a>
  </p>
</footer>
`;
